ckanext/validation_schema_generator/tests/test_action.py

```diff
@@ -143,32 +143,6 @@ def test_apply_resource_must_unapply_pkg_schema(self, table_schema):
         assert pkg[const.PKG_SCHEMA_FIELD]
         assert not pkg['resources'][0][const.RES_SCHEMA_FIELD]
 
-    def test_apply_resource_must_unapply_pkg_schema(self, table_schema):
```

Comment:
@ThrawnCA: why was this test removed?

Reply:
It's a duplicate of the previous test, except with a blank line added.